STYLE_GUIDE.md

```diff
@@ -0,0 +1,57 @@
+**This document is a work in progress.**
+
+# GraphQL Specification Style Guide
+
+This document outlines the styles used in the GraphQL spec to aid editorial and
+consistency. The writing style portions are inspired by the AP style guide. When
+making changes to the GraphQL specification, please aim to be consistent with
+this style guide.
+
+## Auto-formatting
+
+The GraphQL specification is formatted using the `prettier` tool, so you should
+not need to think about gaps between paragraphs and titles, nor about word
```

Comment:
Yes, you shouldn't need to think about it because the tool does it for you. You'd only need to think about it if the tool does something wrong. I don't think there's anything wrong with this text.